app/src/main/java/org/mozilla/focus/home/NavigationOverlayFragment.kt

```diff
@@ -106,6 +93,37 @@ class NavigationOverlayFragment : Fragment() {
         }
     }
 
+    private fun setOnClickListeners(overlay: View) {
+        // We remove the click listeners when dismissing the overlay
+        // so that clicking them doesn't restart the animation.
+        fun removeClickListeners() {
+            overlay.semiOpaqueBackground.setOnClickListener(null)
```

Comment:
Optional nit: I find that, for inner functions, putting the comment inside the function helps more clearly attach it to the inner fun rather than the outer.  I'm not sure if there is an established style guide rule for this, and your mileage may vary.